packages/front-end/components/Features/ExperimentRefSummary.tsx

```diff
@@ -0,0 +1,293 @@
+                  <strong>Temporary Rollout</strong> is enabled. The winning
+                  variation is being served to 100% of users.
+                </>
+              }
+              color="info"
+              experimentId={experiment.id}
+            />
+          </div>
+          <ForceSummary feature={feature} value={releasedValue.value} />
+        </div>
+      );
+    } else {
+      return (
+        <ExperimentSkipped
+          message="This experiment is stopped, but a winner was not selected. This rule will be skipped"
+          experimentId={experiment.id}
+        />
+      );
+    }
+  }
+
+  const hasNamespace = phase.namespace && phase.namespace.enabled;
+  const namespaceRange = hasNamespace
+    ? phase.namespace.range[1] - phase.namespace.range[0]
+    : 1;
+  const effectiveCoverage = namespaceRange * (phase.coverage ?? 1);
+
+  return (
+    <div>
+      {experiment.status === "draft" && (
+        <div className="alert alert-warning">
+          The experiment is in a <strong>draft</strong> state. This rule will be
+          skipped.
+        </div>
+      )}
+      {phase.condition && phase.condition !== "{}" && (
+        <div className="row mb-3 align-items-top">
+          <div className="col-auto">
+            <strong>IF</strong>
+          </div>
+          <div className="col">
+            <ConditionDisplay condition={phase.condition} />
+          </div>
+        </div>
+      )}
+
+      <div className="mb-3 row">
+        <div className="col-auto">
+          <strong>SPLIT</strong>
+        </div>
+        <div className="col-auto">
+          {" "}
+          users by{" "}
+          <span className="mr-1 border px-2 py-1 bg-light rounded">
+            {experiment.hashAttribute || "id"}
+          </span>
+          {hasNamespace && (
+            <>
+              {" "}
+              <span>in the namespace </span>
+              <span className="mr-1 border px-2 py-1 bg-light rounded">
+                {phase.namespace.name}
+              </span>
+            </>
+          )}
+        </div>
+      </div>
+      <div className="mb-3 row">
+        <div className="col-auto">
+          <strong>INCLUDE</strong>
+        </div>
+        <div className="col-auto">
+          <span className="mr-1 border px-2 py-1 bg-light rounded">
+            {percentFormatter.format(effectiveCoverage)}
+          </span>{" "}
+          of users in the experiment
+          {hasNamespace && (
+            <>
+              <span> (</span>
+              <span className="border px-2 py-1 bg-light rounded">
+                {percentFormatter.format(namespaceRange)}
+              </span>{" "}
+              of the namespace and{" "}
+              <span className="border px-2 py-1 bg-light rounded">
+                {/* @ts-expect-error TS(2769) If you come across this, please fix it!: No overload matches this call. */}
+                {percentFormatter.format(coverage)}
```

Comment:
coverage doesn't exist so if you have a namespace you can't load the feature page.